Ubuntu #128781; from SVN; fixes tooltips not disappearing in the systray

--- trunk/gtk/gtktooltip.c	2007/07/25 16:19:36	18543
+++ trunk/gtk/gtktooltip.c	2007/07/25 17:45:22	18544
@@ -1118,7 +1118,7 @@
   /* Hide the tooltip when there's no new tooltip widget */
   if (!has_tooltip_widget)
     {
-      if (current_tooltip && GTK_TOOLTIP_VISIBLE (current_tooltip))
+      if (current_tooltip)
 	gtk_tooltip_hide_tooltip (current_tooltip);
 
       return;
